packages/cloud/src/refreshSpellIconsHandler.ts

```diff
@@ -18,14 +18,18 @@ const gcpCredentials =
 const firestore = new Firestore({
   ignoreUndefinedProperties: true,
   credentials: gcpCredentials,
+  projectId: 'wowarenalogs',
```

Comment:
Without this these libraries will check the local env for a gcp project id and use that instead which can lead to some very confusing errors